include/dsFPD.h

```diff
@@ -105,7 +105,7 @@ dsError_t dsFPInit (void);
  * @brief Sets blink pattern of specified Front Panel Display LED
  * 
  * This function is used to set the individual discrete LED to blink for a specified number of iterations with blink interval.
- * This function must return dsERR_OPERATION_NOT_SUPPORTED if FP State is "OFF". 
+ * This function must return dsERR_OPERATION_NOT_SUPPORTED if FP State is "OFF". To stop the blink, either dsFPSetLEDState() or dsFPTerm() can be invoked.
```

Comment:
What parameter need to dsFPSetLEDState() function , or should the dsSetFPState() function be called with the dsFPD_STATE_OFF parameter?
keep statement in new line

Reply:
As per current dsFPSetLEDState implementation, the LED blinking will be stopped first irrespective of dsFPDLedState_t param

Reply:
Updated